templates/base/store/global/Context.jsx

```diff
@@ -0,0 +1,26 @@
+/** @context.jsx 
+ *  This component is a wrapper which will provide
+ *  the Global Context to other needy components
+ *  */
+
+import React, { useReducer, useEffect } from 'react';
+import reducer, { initialState } from './reducer';
+import { APP_LOADED } from './actions';
+
+// Creating global context
+export const Context = React.createContext();
+
+let [state, dispatch] = useReducer(reducer, initialState);
```

Comment:
This is context definition file, here we will not execute actions and useReducer

I think you miss understood my comment on uses of these actions that was from reducer uses as it was empty